Test TestGetIndices fails when running in Europe/Moscow timezone

I run test on my local machine with Europe/Moscow timezone and it fails
```bash
$ go test github.com/jaegertracing/jaeger/plugin/storage/es/dependencystore -run TestGetIndices
--- FAIL: TestGetIndices (0.00s)
        Error Trace:    storage_test.go:200
	Error:      	Not equal:
	            	expected: []string{"jaeger-dependencies-1995-04-21"}
	            	actual: []string{"jaeger-dependencies-1995-04-21", "jaeger-dependencies-1995-04-20"}

	            	Diff:
	            	--- Expected
	            	+++ Actual
	            	@@ -1,3 +1,4 @@
	            	-([]string) (len=1) {
	            	- (string) (len=30) "jaeger-dependencies-1995-04-21"
	            	+([]string) (len=2) {
	            	+ (string) (len=30) "jaeger-dependencies-1995-04-21",
	            	+ (string) (len=30) "jaeger-dependencies-1995-04-20"
	            	 }
FAIL
FAIL	github.com/jaegertracing/jaeger/plugin/storage/es/dependencystore	0.011s
```

--- a/plugin/storage/es/dependencystore/storage_test.go
+++ b/plugin/storage/es/dependencystore/storage_test.go
@@ -174,7 +174,7 @@ func createSearchResult(dependencyLink string) *elastic.SearchResult {
 }
 
 func TestGetIndices(t *testing.T) {
-	fixedTime := time.Date(1995, time.April, 21, 4, 12, 19, 95, time.Local)
+	fixedTime := time.Date(1995, time.April, 21, 4, 12, 19, 95, time.UTC)
 	testCases := []struct {
 		expected []string
 		lookback time.Duration

